tests/playTests.sh

```diff
@@ -485,6 +485,11 @@ $ZSTD -bi0 --fast tmp1
 $ECHO "with recursive and quiet modes"
 $ZSTD -rqi1b1e2 tmp1
 
+$ECHO "\n===>  zstd compatibility tests "
+
+./datagen > tmp
+$ZSTD --format=zstd tmp 2> tmplog
```

Comment:
nit: you want to add `-f` command, in case `tmp.zst` already exist in current directory, which would trigger `$ZSTD` to request an overwrite confirmation.